content/httpd/ros.apache.conf.inc

```diff
@@ -1,3 +1,3 @@
-<% migrated_ros_posts.each do |post| %>
+<% @items.find_all('/question/*.md').each do |post| %>
```

Comment:
This would create invalid rewrites for questions that did not migrate. I think we need to something like this to make sure we don't render a redirect for non-migrated posts.
```suggestion
<% @items.find_all('/question/*.md').select{|post| post[:migrated_url]}.each do |post| %>
```